worlds/v6/__init__.py

```diff
@@ -49,12 +49,14 @@ def set_rules(self):
         self.area_cost_map = {}
         set_rules(self.multiworld, self.options, self.player, self.area_connections, self.area_cost_map)
 
-    def create_item(self, name: str) -> Item:
-        return V6Item(name, ItemClassification.progression, item_table[name], self.player)
+    def create_item(self, name: str, classification: ItemClassification) -> Item:
```

Comment:
This is a function that's overridden from the superclass (World), and your new function signature is incompatible with the one from the superclass.

You can fix this by supplying a default value for `classification`, which should ideally be "filler":

`def create_item(self, name: str, classification: ItemClassification = ItemClassification.filler) -> Item:`

Reply:
Got it, I'll add that in tonight.